src/main/java/edu/ie3/datamodel/models/UniqueEntity.java

```diff
@@ -10,7 +10,10 @@
 
 /** Supplies every subclass with a generated UUID, making it unique */
 public abstract class UniqueEntity {
-  private final UUID uuid;
+  /** Field name of {@link UniqueEntity} uuid */
+  public static final String UUID_FIELD_NAME = "uuid";
+
+  protected final UUID uuid;
```

Comment:
![image](https://user-images.githubusercontent.com/42348110/78894432-512aba00-7a6d-11ea-80c2-0c7fc3b4e18f.png)
```suggestion
  private final UUID uuid;
```